### PR TITLE
add seealso section for default template

### PR DESCRIPTION
**Pull request by ansyral**
#383

### CODE REVIEW

File: src/docfx.website.themes/common/common.js [hunk 1]
```diff
@@ -118,3 +119,30 @@ function getOverrideTemplate(uid) {
...
+    if (model.children) {
+        model.children.forEach(function (c) {
+            if (c.seealso) {
+                for(var key in c.seealso) {
+                    addIsCref(c.seealso[key]);
+                    seealso.push(c.seealso[key]);
+                }
+            }
+        });
+    }
```
Comment (superyyrrzz) [inline src/docfx.website.themes/common/common.js]: remove extra spaces

File: src/docfx.website.themes/default/partials/class.tmpl.partial [hunk 2]
```diff
@@ -197,3 +197,18 @@
   {{/definition}}
 </div>
 {{/extensionMethods}}
+{{#seealso.0}}
+<h3 id="seealso">{{__global.seealso}}</h3>
+<div class="seealso">
+{{/seealso.0}}
+{{#seealso}}
+    {{#isCref}}
```
Comment (superyyrrzz) [inline src/docfx.website.themes/default/partials/class.tmpl.partial]: minor: 2 spaces

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by vwxyzh** [comment]
👍

**Pull request merged**